src/css/style.css

```diff
@@ -222,6 +222,18 @@ nav ol {
   display: flex;
 }
 
+nav ol.secondary-nav {
```

Comment:
This is really good design instincts, I really like how this looks, and how simple you kept it.

I actually feel it looks fine without setting the font size smaller. Even the font size being the same in desktop / mobile looks pretty good to me:

![2023-04-28_17-19](https://user-images.githubusercontent.com/8533113/235109996-0d76312e-f209-47ff-bf1d-76ceb5577d25.png)

There's probably something to be said about making there be more of a separation between the main nav and these sub-navs, but i think that's a separate ticket for main nav itself. for now I'm pretty happy with this.

Ah, the other thing is, if we don't need to bother setting font-size per screen width, you can take all this out of the `@media` sections and just define the style once somewhere, which is nice and DRY